### PR TITLE
sap_vm_provision: exec flow amendments from testing

### PR DESCRIPTION
**Pull request by sean-freeman**
sap_vm_provision: exec flow amendments from testing

### CODE REVIEW

**Review by marcelmamula — APPROVED** [review]
LGTM after connectivity typo is resolved @sean-freeman I suppose removal of AWS and GCP from Readme is because Ansible collections are used instead of CLI, right?

**Comment by berndfinger** [comment]
> LGTM after connectivity typo is resolved @sean-freeman I suppose removal of AWS and GCP from Readme is because Ansible collections are used instead of CLI, right?

@marcelmamula As there has been no reply yet from @sean-freeman, should we just assume that the answer to your question is `yes`, and just merge and fix the spelling error and the ansible-lint error in a new small PR right after merging?

File: roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_powervs/execute_main.yml
Comment (Wabri) [inline roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_powervs/execute_main.yml:376]:
```suggestion
    # required for both outbound Public Internet connectivity and internal traffic to other IBM Cloud Services
```

File: roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_vs/execute_setup_ha.yml
Comment (Wabri) [inline roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_vs/execute_setup_ha.yml:1249]:
```suggestion
```

Too many blank line => https://github.com/sap-linuxlab/community.sap_infrastructure/actions/runs/10631453083/job/29472374510?pr=58

**Review by Wabri — CHANGES_REQUESTED** [review]
Minor linting problem to fix

**Pull request merged**
